gridappsd/goss.py

```diff
@@ -89,6 +89,7 @@ def __init__(self, username=None, password=None,
         self._topic_set = set()
         self._override_thread_fc = override_threading
         self._router_callback = CallbackRouter()
+        self.result_format = None
```

Comment:
Do we want to default to 'JSON' or require it to be non-json unless specified?

Reply:
We are defaulting to JSON as all APIs respond back in JSON unless specified otherwise.